x-pack/test/functional/services/ml/api.ts

```diff
@@ -1347,6 +1347,14 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       log.debug('> Trained model deleted');
     },
 
+    async deleteTrainedModelInferencePipeline(pipelineId: string) {
+      log.debug(`Deleting inference pipeline with id "${pipelineId}"`);
+      const { body, status } = await esSupertest.delete(`/_ingest/pipeline/${pipelineId}`);
+      this.assertResponseStatusCode(200, status, body);
+
+      log.debug('> Inference pipeline deleted');
+    },
+
```

Comment:
we already have this method defined [here](https://github.com/elastic/kibana/blob/d61c2541096d0ce11551f9caaf161d047047caf1/x-pack/test/functional/services/ml/api.ts#L1501) 
```suggestion
```